src/app/views/query-runner/QueryRunner.tsx

```diff
@@ -85,10 +80,23 @@ export class QueryRunner extends Component<
     }
   };
 
-  public render() {
-    const { httpMethods } = this.state;
-    const { graphExplorerMode, isLoadingData } = this.props;
+  private handleOnVersionChange = (option?: IDropdownOption) => {
+    const { sampleQuery } = this.props;
+    if (option) {
```

Comment:
Rename this to a more descriptive name.

Variables should describe the data they hold.

for example, urlVersion